examples/SmartShop/contracts/BuyerContract.aes

```diff
@@ -1,6 +1,6 @@
 contract SellerInterface =
-  payable stateful entrypoint received_item : () => bool
-  entrypoint get_contract_address : () => address
+  entrypoint received_item : () => unit
+  entrypoint seller_contract_address : () => address
```

Comment:
The function `seller_contract_address` is completely useless...

You would call it like `seller.seller_contract_address()` and then you can just as well just say `seller.address` which is shorter and doesn't involve a remote contract call.

Reply:
Fixed in the [b182f04 ](https://github.com/aeternity/aepp-sophia-examples/pull/58/commits/b182f04aae62e33f0bdcc7793d954fd996983bd4) commit.